app/routers.py

```diff
@@ -81,7 +82,9 @@ def request_validation_error_handler(request: Request, e: RequestValidationError
                 # prepend last entry in error location if available, skip global and leading context
                 error_message = f'{error_location[-1]}: {error_message}'
 
-            errors.append({'error': 'ValidationError', 'message': error_message})
+            if error_message not in seen_messages:
+                errors.append({'error': 'ValidationError', 'message': error_message})
+                seen_messages.add(error_message)
```

Comment:
I can't find any tests associated with this. Please include at least one test for this.